CHANGELOG.md

```diff
@@ -1,3 +1,14 @@
+## 0.2.0-beta2
+
+- `@clickhouse/client-browser` renamed to `@clickhouse/client-web` as it makes more sense,
+  considering the supported platforms.
+- Firefox is now officially supported with `@clickhouse/client-web`.
+
+### Breaking changes (both Node.js/Web)
```

Comment:
We should combine all the breaking changes in a single release note for `0.2.0` later.